androidApp/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,11 @@
 <resources>
-    <string name="app_name">Press</string>
+  <string name="app_name">Press</string>
+
+  <item name="homeViewId" type="id" />
```

Comment:
I don't mind resources named in camel case, but the project has used underscores so far, can you follow the naming for consistency?

Reply:
Also, the the `Id` suffix is redundant. Its usages will already indicate that it's an ID: `R.id.homeViewId`.

Reply:
I see. Lately I just drop Kotlin Android Extension (by which if your Id is in snake_case your code will look terrible ...) to ViewBinding (it automatically converts the snake_case id to camelCase). So some habits are still there... Will fix the styling here.